src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,67 @@
+import classNames from 'classnames';
+import React, { MouseEvent, useContext } from 'react';
+
+import { FilterContext } from '../../context/FilterContext';
+import { FilterType } from '../../types/SortType';
+
+export const Filter: React.FC = React.memo(
+  () => {
+    const { filter, setFilter } = useContext(FilterContext);
+
+    const onFilterChange = (value: string) => {
+      switch (value) {
+        case FilterType.ACTIVE:
+          setFilter(FilterType.ACTIVE);
+          break;
+
+        case FilterType.COMPLETED:
+          setFilter(FilterType.COMPLETED);
+          break;
+
+        default:
+          setFilter(FilterType.ALL);
+      }
+    };
+
+    const handlerOnClick = (event: MouseEvent<HTMLAnchorElement>) => {
```

Comment:
```suggestion
    const handleClick = (event: MouseEvent<HTMLAnchorElement>) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364